### DISCUSSION TITLE
Allow overriding the hitsound volume that is set by the map creator.

### DISCUSSION THREAD

**Discussion by UMTTEPATOP** [ppy/osu#28971] [Ideas]
There are some maps that I can't play due to volume of hitsounds being too low, and constantly changing the volume is very unpleasant, so it would be nice if there was an option to always override the hitsound volume according to the % volume you have set in settings, rather than combine both your settings and in-map volume.

**Comment by MizDaMeaner** [comment]
^This pls! some also change the volume of the hitsound during the map which makes it impossible for me, so having the option to keep the volume the same throughout a map would be extremely helpful too.

**Comment by Akiraa888** [comment]
Need this ! Pls add this option to the game

I think make it a mod? similar to MuteMod

![2](https://github.com/user-attachments/assets/3c2d2252-b14b-460d-98da-9a11479fa88e)

**Comment by Doksha** [comment]
Mania is literally impossible without this. So many braindead mappers that make some hitsounds very quiet, and the lazer editor cant handle editing hitsounds on a large selection of notes.
It is so annoying only being able to play some arbitrary selection of maps because other maps don't have hitsounds

**Comment by Alsen99** [comment]
I agree, I want to mute the custom hitsounds mappers make
